docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+
+<p class="blog-description">1492 out of 1589 answered</p>
+
+As the MUI Core v5 release is still relatively recent, it's still expected to see a great number of developers using JSS, and there will always probably be some of you who use it because of pure preference.
+
+However, it’s notable the styled-components and emotion growth, and that might be because it’s what our new styling solution is based off. We're still seeing a lot of questions and also opportunities to make it better, so this year we'll focus on expanding and refining [MUI System](https://mui.com/system/basics/) even further.
+
+### Did you recently migrate to MUI Core v5?
+
+<img src="/static/blog/2021-survey/23.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Yes: 63.3%; No: 36.7%" />
+
+<p class="blog-description">1546 out of 1589 answered</p>
+
+Even though a lot of you who answered the survey has migrated, that are still a lot more to convince migrating. 2022 is going to be a lot focused in that.
+
+### What's the statement that most defines your migration experience?
+
+<img src="/static/blog/2021-survey/24.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="It was tricky but MUI's documentation and resources helped me to get it right: 43%; It was smooth and it is now working fine: 38.5%; It wasn't good, I had many problems and it took me a long time to finish: 14.3%; It was bad and I ended up regretting it: 2.8%; Other: 1.4%" />
+
+<p class="blog-description">930 out of 1589 answered</p>
+
+### What could MUI do to improve the migration experience?
+
+<img src="/static/blog/2021-survey/25.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="General documentation improvements: 43.2%; Automation: 20.7%: Fewer breaking changes: 15.6%; Fewer styling soluiton breaking changes: 8.5%; More tutorials: 5.4%; More styling solution migration tutorial: 4.4%; Migrate Core and X together: 1%" />
+
+<p class="blog-description">472 out of 1589 answered</p>
+
+- **General documentation improvement:** Documentation plays a huge part in defining the developer experience. We definitely need to explore more ways of making the step-by-step easier, add more examples, refine the copywriting, and maybe overall make it more beginner-friendly.
+  - There are other topics converging with this one, like requests for more tutorials.
+- **Automation:** We prepared codemods to aid the v5 migration, given the amount of changes it had. We've seen positive feedback for it, which is encouraging since it's our first time providing tools to automate a migration process. However, it is becoming clear how and when the codemods weren't too effective, so we'll make sure to refine it more.
```

Comment:
```suggestion
- **Automation:** We prepared codemods to help with the v5 migration, given the number of changes. This has been met with positive feedback, which is encouraging since it's our first time providing tools to automate the migration process. However, it is becoming clear how and when the codemods weren't too effective, so we'll make sure to refine it more.
```